wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -324,22 +324,39 @@ safe) semantics.
    * Whether the command is currently grouped in a command group.  Used as extra
    * insurance to prevent accidental independent use of grouped commands.
    */
-  bool IsGrouped() const;
+  bool IsComposed() const;
 
   /**
-   * Sets whether the command is currently grouped in a command group.  Can be
-   * used to "reclaim" a command if a group is no longer going to use it.  NOT
+   * Sets whether the command is currently composed in a command composition.  Can be
+   * used to "reclaim" a command if a composition is no longer going to use it.  NOT
    * ADVISED!
    */
-  void SetGrouped(bool grouped);
+  void SetComposed(bool isComposed);
+
+  /**
+   * Whether the command is currently composed in a command group.  Used as extra
+   * insurance to prevent accidental independent use of grouped commands.
+   */
+  bool IsComposed() const;
 
   /**
-   * Whether the given command should run when the robot is disabled.  Override
-   * to return true if the command should run when disabled.
+   * Whether the command is currently grouped in a command group.  Used as extra
+   * insurance to prevent accidental independent use of grouped commands.
    *
-   * @return whether the command should run when the robot is disabled
+   * @deprecated Moved to IsComposed()
    */
-  virtual bool RunsWhenDisabled() const { return false; }
```

Comment:
Was runsWhenDisabled supposed to be removed?

Reply:
No, I have no idea how that happened